agent/ecscni/plugin_windows.go

```diff
@@ -52,11 +79,17 @@ func (client *cniClient) setupNS(ctx context.Context, cfg *Config) (*current.Res
 			cniNetworkConfig.Network.Type,
 			cfg.ContainerID)
 		runtimeConfig.IfName = networkConfig.IfName
-		_, err := client.libcni.AddNetwork(ctx, cniNetworkConfig, &runtimeConfig)
+		result, err := client.libcni.AddNetwork(ctx, cniNetworkConfig, &runtimeConfig)
 		if err != nil {
 			return nil, errors.Wrap(err, "add network failed")
 		}
 
+		// We save the result from ecs-bridge setup invocation of the plugin.
+		if strings.EqualFold(ECSBridgeNetworkName, cniNetworkConfig.Network.Name) &&
```

Comment:
curious as to why is `ECSBridgeNetworkName` comparison case insensitive whereas `ECSVPCENIPluginExecutable` is not?

Reply:
This is because `ECSBridgeNetworkName` is the name of the default network on Windows instances which is to be used as the `ecs-bridge`. The name of this network can be case insensitive.
On the other hand, the plugin executable cannot be case insensitive otherwise it will lead to an error.